tensorrt_llm/_torch/attention_backend/sparse/kernels/__init__.py

```
@@ -0,0 +1,15 @@
+    triton_index_gather,
+    triton_softmax,
+    triton_topk,
+)
+
+__all__ = [
+    "triton_index_gather",
+    "triton_bmm",
+    "triton_softmax",
+    "triton_flatten_to_batch",
+    "triton_topk",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)